en/computer_vision/safe_landing.md

```diff
@@ -0,0 +1,49 @@
+# Safe Landing
+
+The *Safe Landing* feature ensures that vehicles only land on flat terrain.
+
+The feature can be enabled in both [Land mode](../flight_modes/land.md) and [Mission mode](../flight_modes/mission.md) on multicopter vehicles (and VTOL vehicles in MC mode) that have a companion computer running the appropriate vision software.
+
+If commanded to land, the vehicle first descends to a height where it can measure the surface (companion computer `loiter_height` parameter).
+If the landing area is not sufficiently flat, the vehicle moves outwards in a square-spiral pattern, periodically stopping to re-check the terrain for a landing spot that isn't too rough.
+
+> **Warning** The feature will not prevent a vehicle from landing on a road (even if a car is detected it will be "forgotten" once it moves past).
+  It will prevent landing on water, as the cameras used do not "see" the water surface at all.
```

Comment:
FYI, this came up at dev summit, and this was Julian Kent's response. Is it true? I mean it is true for our setup, but will it be true in future if someone used a different camera?

Reply:
This is true for stereo cameras. They won't generate points on water that is still due to there being no distinguishing features to do the triangulation with. And water that is moving enough to have nice features won't be flat enough for the safe landing planner.

Reply:
Radar or ultrasound might get returns on still water though. LIDAR shouldn't.

Reply:
Is it actually true that no features will prevent landing? Or will it cause a descent to find features, effectively resulting in a water landing?

Reply:
No features will prevent landing

Reply:
@mrivi Just FMI (further to @ndepal 's point), what I think will happen there is that the vehicle will descend to `loiter_height`. It won't detect any features with a stereo camera, so it will decide that it doesn't have enough information to land. Presumably at this point it will start doing its spiral?
Does it send any setpoints to the vehicle at this point - or just heartbeats?

Reply:
@jkflying Thanks. I have moved the clarification on water landing into a "clarifications section".

Reply:
Yes, that's correct. In order to do the spiral we send setpoints to the FCU.